manageiq-operator/Gopkg.toml

```diff
@@ -0,0 +1,74 @@
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
+required = [
```

Comment:
Why not use go mod?

Reply:
👍 @gyliu513 is this something that can be done in a follow up or is this something that's hard to change later?

Reply:
@Fryguy it is simple to change, we can merge it first and use go mod in follow up PRs :)

Reply:
It seems that the previous way enforces committing all of the dependencies which affects the git history.  Given that it's a *lot* of code, I'd prefer we use go mod from the start.

Reply:
Given that we can commit without the dependencies, I'm inclined to merge as is, and do go mod as a follow up.